intro_to_python/intro_to_python.md

```diff
@@ -2,20 +2,21 @@
 
 author:   Meredith Lee
 email:    leemc@chop.edu
-version:  1.0.0
+version:  2.0.0
+module_template_version: 2.0.0
 language: en
 narrator: UK English Female
 title: Introduction to Python
-comment:  This module introduces the Python programming language, explores why Python is useful in research, and demonstrates how to write Python code in a Jupyter notebook (and why you might want to).
-long_description: Python is a versatile programming language that is frequently used for data analysis, machine learning, web development, and more. If are interested in learning about any of these topics and are new to programming, are joining a team where Python is used, or have just heard about Python and are looking for a general introduction to the language, this module is a good place to start. This is appropriate for someone at the beginner level.
-
+comment:  This module introduces the Python programming language, explores why Python is useful in research, and describes how to download Python and Jupyter.
+long_description: Python is a versatile programming language that is frequently used for data analysis, machine learning, web development, and more. If you are interested in learning about any of these topics and are new to programming, are joining a team where Python is used, or have just heard about Python and are looking for a general introduction to the language, this module is a good place to start. This is appropriate for someone at the beginner level.
+estimated_time: 20m
 @learning_objectives  
 
 After completion of this module, learners will be able to:
 
 - Describe what Python is and why they might want to use it for research
-- Recognize Python code
-- Write text and simple code in a Jupyter notebook
+- Identify several ways to write Python code
+- Understand the purpose and utility of a Jupyter notebook
 - Download Python to their own computers
```

Comment:
this needs to be updated to either also include google colab info on the same objective or add it as a fifth objective 
(QA item: "All major topics covered are represented by a learning objective -- there's no significant knowledge imparted that isn't specified in the learning objectives" )

Reply:
actually, looking back at the formative assessment section, every LO needs to have at least one Q of its own, so I think that download python/colab would need to be a singular LO unless you want to add another Q (which tbh seems like overkill to me)

Reply:
actually, looking back at the formative assessment section, every LO needs to have at least one Q of its own, so I think that download python/colab would need to be a singular LO unless you want to add another Q (which tbh seems like overkill to me)